server/api.go

```diff
@@ -254,6 +254,36 @@ func (d dexAPI) ListPasswords(ctx context.Context, req *api.ListPasswordReq) (*a
 
 }
 
+func (d dexAPI) VerifyPassword(ctx context.Context, req *api.VerifyPasswordReq) (*api.VerifyPasswordResp, error) {
+	if req.Email == "" {
+		return nil, errors.New("no email supplied")
```

Comment:
This being GRPC, can we return `status.Error(codes.InvalidArguments, "no email supplied")` or something like that? `errors.New()`, or, any error would translate to `codes.Internal` or `codes.Unknown`, can't remember.

Reply:
I can change the error returned, but that will be not consistent with the rest of errors returned by other api. Do you want me to do it anyway ? and you will have some other tasks where you will change the other functions ?

Reply:
I'll deal with the others. :smiley: You know what, you're correct, this is fine, let's keep them as-is and we can update all the errors in one go later.